### PR TITLE
Disable Developer > Assets > ATP Asset Migration menu item

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
Testing

The Developer > Assets > ATP Asset Migration menu item no longer is present; not on desktop and not in HMD.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 38bb48e72c41ad30bd825f2aa48f6ff89110aa74 available at: https://gist.github.com/d31eb78fb2ac25ccb8dd3d8ae9045f20<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9939-38bb48e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9939-38bb48e.dmg

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -417,13 +417,16 @@ Menu::Menu() {
     }
 
     // Developer > Assets >>>
+    // Menu item is not currently needed but code should be kept in case it proves useful again at some stage.
+    /*
```
Comment (ZappoMan) [inline interface/src/Menu.cpp]: Can we do this as `#ifdef` instead?

Comment (ctrlaltdavid) [inline interface/src/Menu.cpp]: Sure, but #ifdef on what / where?

**Comment by hifi-gustavo** [comment]
Build console output for commit: ab7d82edb6ba5f9a32770afd8cc941f639f6230c available at: https://gist.github.com/d31eb78fb2ac25ccb8dd3d8ae9045f20<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9939-ab7d82e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9939-ab7d82e.dmg

**Comment by Nex-Pro** [comment]
checking this

**Comment by Nex-Pro** [comment]
### PASS

Compared this to Dev:
  Developer > Assets  menu item is now gone
as of course is Developer > Assets > ATP Asset Migration

**Comment by ctrlaltdavid** [comment]
Yes, Developer > Assets menu intentionally removed.

**Pull request merged**